index.js

```diff
@@ -313,6 +315,17 @@ function Argv (processArgs, cwd) {
     return options
   }
 
+  var groups = {'Options:': []}
+  self.group = function (opts, groupName) {
+    if (!Array.isArray(opts)) opts = [opts]
+    if (!groups[groupName]) groups[groupName] = []
+    Array.prototype.push.apply(groups[groupName], opts)
```

Comment:
Not that it helps readability or efficiency any, but these 3 lines could be reduced to:

```js
groups[groupName] = (groups[groupName] || []).concat([].concat(opts))
```

Feel free to ignore this. :grinning: